diff -Naur partysip-0.6.0.orig/conf/Makefile.am partysip-0.6.0/conf/Makefile.am
--- partysip-0.6.0.orig/conf/Makefile.am	Thu Jun 26 20:39:46 2003
+++ partysip-0.6.0/conf/Makefile.am	Thu Jun 26 20:45:06 2003
@@ -3,23 +3,8 @@
 pkgdatadir = $(prefix)/$(PACKAGE)
 
 install-data-local:
-	$(mkinstalldirs) $(CONFIG_DIR);
-	@chmod 0755 $(CONFIG_DIR);
+	$(mkinstalldirs) $(DESTDIR)/$(CONFIG_DIR);
 	@echo "Installing admin files : partysip.conf";
-	@if test -f $(CONFIG_DIR)/partysip.conf; then \
-	  echo "********************************************************"; \
-	  echo "**** File partysip.conf has already been installed. ****"; \
-	  echo "****   MERGE CHANGES MANUALLY in partysip.conf.     ****"; \
-	  echo "********************************************************"; \
-	  echo "Installing new admin file: partysip.conf.$(PSP_VERSION)";  \
-	  echo "********************************************************"; \
-	  $(INSTALL_DATA) $(top_srcdir)/conf/partysip.conf $(CONFIG_DIR)/partysip.conf.$(PSP_VERSION); \
-	else \
-	  echo "********************************************************"; \
-	  echo "****         First installation detected.           ****"; \
-	  echo "****      Please, edit the file partysip.conf.      ****"; \
-	  echo "********************************************************"; \
-	  $(INSTALL_DATA) $(top_srcdir)/conf/partysip.conf $(CONFIG_DIR)/; \
-	fi
+	$(INSTALL_DATA) $(top_srcdir)/conf/partysip.conf $(DESTDIR)/$(CONFIG_DIR)/;
 
 
