--- ./conf/Makefile.am.org	Sat Dec 28 01:50:58 2002
+++ ./conf/Makefile.am	Thu Jan  2 09:59:46 2003
@@ -1,27 +1,25 @@
 EXTRA_DIST = partysip.conf
 
-sysconfdir = $(prefix)/etc/partysip/
-
 pkgdatadir = $(prefix)/$(PACKAGE)
 
 install-data-local:
-	$(mkinstalldirs) $(sysconfdir);
-	@chmod 0755 $(sysconfdir);
+	$(mkinstalldirs) $(DESTDIR)$(sysconfdir);
+	@chmod 0755 $(DESTDIR)$(sysconfdir);
 	@echo "Installing admin files : partysip.conf";
-	@if test -f $(sysconfdir)partysip.conf; then \
+	@if test -f $(DESTDIR)$(sysconfdir)partysip.conf; then \
 	  echo "********************************************************"; \
 	  echo "**** File partysip.conf has already been installed. ****"; \
 	  echo "****   MERGE CHANGES MANUALLY in partysip.conf.     ****"; \
 	  echo "********************************************************"; \
 	  echo "Installing new admin file: partysip.conf.$(PSP_VERSION)";  \
 	  echo "********************************************************"; \
-	  $(INSTALL_DATA) $(top_srcdir)/conf/partysip.conf $(sysconfdir)partysip.conf.$(PSP_VERSION); \
+	  $(INSTALL_DATA) $(top_srcdir)/conf/partysip.conf $(DESTDIR)$(sysconfdir)partysip.conf.$(PSP_VERSION); \
 	else \
 	  echo "********************************************************"; \
 	  echo "****         First installation detected.           ****"; \
 	  echo "****      Please, edit the file partysip.conf.      ****"; \
 	  echo "********************************************************"; \
-	  $(INSTALL_DATA) $(top_srcdir)/conf/partysip.conf $(sysconfdir); \
+	  $(INSTALL_DATA) $(top_srcdir)/conf/partysip.conf $(DESTDIR)$(sysconfdir); \
 	fi
 
 
